src/App.tsx

```diff
@@ -15,81 +29,161 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
+function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+): string [] {
+  // To avoid the original array mutation
+
+  const visible = [...goods];
+
+  visible.sort((word1, word2) => {
+    switch (sortType) {
+      case 2:
```

Comment:
You have a enum for it
```suggestion
      case sortType. LENGTH:
```